crates/library/std/src/evm.fe

```diff
@@ -236,8 +238,8 @@ pub unsafe fn call_data_size() -> u256 {
     return __calldatasize()
 }
 
-pub unsafe fn call_data_copy(to_offset t: u256, from_offset f: u256, len: u256) {
-    __calldatacopy(t, f, len)
+pub fn call_data_copy(buffer: buf::MemoryBuffer) {
+    unsafe { __calldatacopy(buffer.offset(), buffer.offset(), buffer.len()) }
```

Comment:
the `from_offset` param should remain and be used as the second parameter in `__calldatacopy`, as it is the offset in calldata that we are copying from

https://docs.soliditylang.org/en/latest/yul.html#low-level-functions